main/views.py

```diff
@@ -126,4 +163,4 @@ def api_match_detail(request, match_id):
 @require_POST
 def api_remove_invalid_matches(request):
     response = remove_invalid_matches()
-    return HttpResponse("The list of invalid ids removed is: " + str(response))
+    return JsonResponse(json.dumps(response), safe=False)
```

Comment:
I thought it's either
`JsonResponse(dictionary_data)`
or
`HttpResponse(json.dumps(anything), content_type='application/json')`
I suggest you go with the first solution, you just have to envelope your list in a dict:
```Python
removed_match_ids = remove_invalid_matches()
return JsonReponse({'removed_match_ids': removed_match_ids})
```
What do you think?

Reply:
hmm that looks good, I will do that